src/main/java/frc/trigon/robot/subsystems/turret/simulationturret/SimulationTurretConstants.java

```diff
@@ -0,0 +1,37 @@
+package frc.trigon.robot.subsystems.turret.simulationturret;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+
+public class SimulationTurretConstants {
+    static final double VOLTAGE_COMPENSATION_SATURATION = 12;
+
+    private static final double
+            MAX_MOTOR_VELOCITY = 200,
+            MAX_MOTOR_ACCELERATION = 3;
```

Comment:
3 vs 200 is a big units difference

Reply:
I don't know what a good number for it is

Reply:
Just a number that's close to the maximum velocity 
Because the ratio between them is too big, and this is in degrees per second iirc?

Reply:
But are there numbers that are normally around it or is it very different for everything?